jupyterhub_singleuser_profiles/ui/src/App/StartServerModal.tsx

```diff
@@ -0,0 +1,302 @@
+import * as React from 'react';
+import classNames from 'classnames';
+import {
+  Accordion,
+  AccordionItem,
+  AccordionToggle,
+  AccordionContent,
+  Alert,
+  Button,
+  Modal,
+  ModalVariant,
+  Progress,
+  ProgressVariant,
+} from '@patternfly/react-core';
+import { APIGet } from '../utils/APICalls';
+import { HubUserRequest } from '../utils/HubCalls';
+import { useWatchSpawnProgress } from '../utils/useWatchSpawnProgress';
+import { CM_PATH, DEV_MODE, FOR_USER, USER } from '../utils/const';
+import { UserConfigMapType } from '../utils/types';
+
+import './StartServerModal.scss';
+
+type StartServerModalProps = {
```

Comment:
Do we need to an export here as well?

Reply:
Not at this time. Potentially if we add unit tests it _might_ be necessary but we can do that when we need to.